akka-http-core/src/main/scala/akka/http/impl/engine/ws/WebSocketClientBlueprint.scala

```diff
@@ -8,26 +8,23 @@ import akka.NotUsed
 import akka.http.scaladsl.model.ws._
 
 import scala.concurrent.{ Future, Promise }
-
```

Comment:
Either IntelliJ or scalariform had a field day on the imports, it seems.  Let me know if you want me to revert any of these lines :)